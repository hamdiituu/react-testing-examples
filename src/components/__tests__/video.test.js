import React from "react";
import renderer from "react-test-renderer";
import Videos from "../videos";

test("Videos Empty Renders : Video Not Found!", () => {
  const tree = renderer.create(<Videos />).toJSON();
  expect(tree).toMatchSnapshot();
});

test("Videos List Renders", () => {
  const testList = ["Udemy", "Video", "Snappy"];
  const tree = renderer.create(<Videos videoList={testList} />).toJSON();
  expect(tree).toMatchSnapshot();
});
