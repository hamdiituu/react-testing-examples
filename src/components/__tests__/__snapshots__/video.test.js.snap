// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Videos Empty Renders : Video Not Found! 1`] = `
<h3>
  Video Not Found!
</h3>
`;

exports[`Videos List Renders 1`] = `
<ul>
  <li>
    Udemy
  </li>
  <li>
    Video
  </li>
  <li>
    Snappy
  </li>
</ul>
`;
